src/js/sentry.js

```diff
@@ -0,0 +1,44 @@
+import * as Sentry from '@sentry/browser';
+
+const API_KEY = 'https://80e5a70255df4bd3ba6eb3b4bfebc58c@sentry.io/1466891';
```

Comment:
Is it OK for this to be in the code?

Reply:
Not sure. We're exposing the api key for the pendo stuff in `analytics.js`, so I thought it should be fine? We can always wrap it up and do it somewhere else.

Reply:
If it's already exposed, I guess we might as well do the same here. I'm not sure how sensitive the api key is, but if it is, we'll probably want to look at injecting it for both sentry and pendo.